cmd/crdvalidator/handlers/crd.go

```diff
@@ -73,3 +77,11 @@ func (cv *CrdValidator) InjectDecoder(d *admission.Decoder) error {
 	cv.decoder = d
 	return nil
 }
+
+// enabled takes a CRD and checks its content to see crdvalidator
+// is disabled explicitly
+func enabled(crd *apiextensionsv1.CustomResourceDefinition) bool {
```

Comment:
based on the description of the function, I would expect this code to look for
```
crd.GetAnnotations()[annotation.ValidationKey] == annotation.Disabled 
```


I suggest a change to the comment or the implementation. I think changing this function to be called `disabled()` and checking for the specific disabled condition makes the most sense, if we want to be strict about in what cases CRD validation is disabled.